setup.py

```diff
@@ -42,6 +42,7 @@
     "google-resumable-media >= 0.6.0, < 3.0dev",
     "packaging >= 14.3",
     "protobuf >= 3.12.0",
+    "python-dateutil >= 2.7.0, <3.0dev",
```

Comment:
Let's maybe use [2.7.2](https://github.com/dateutil/dateutil/releases/tag/2.7.2) as a minimum version, which has been released 14 days after `2.7.0` and includes a fix for the setup script in non-unicode environments? Or the current most recent 2.7.x release with even more fixes and `tzdata` updates?

(didn't test if any of those affect this code, though, just went through the release notes)

Reply:
Sounds good. I could see those issues biting us.